### ISSUE TITLE
[Enhancement] Adjust Featured Assets

### ISSUE THREAD

**Issue by moritzkirstein** [deltaDAO/mvg-portal#50]
## Motivation / Problem
The Featured Assets section should become more of an intro to different use-cases that are already implemented in various demos.

## Solution 
- [ ] change the section title to "Featured Use Cases"
- [ ] Adjust featured assets config
- [ ] Adjust names of assets to reflect the specific use case
- [ ] Change "All data sets and algorithms" to "All Data Services"

**Comment by LucaMilanese90** [comment]
@moritzkirstein when you say "Adjust featured assets config" do you mean adjust functions naming to point at use cases instead of assets or do you have something else in mind? 
Also I'm not sure I'll be able to proceed with the renaming of the assets.

**Comment by moritzkirstein** [comment]
Yes. We first need to agree on the assets / use-cases we want to choose here and settle on names.
We then need to cahnge the `featured` section in the `address.config` accordingly.

I'll update the issue / comment again, once that is decided

**Comment by moritzkirstein** [comment]
The newly published assets are now available:
https://portal.minimal-gaia-x.eu/asset/did:op:E4bE4AF489089bE4c199149A9f9D9C1fbEa138A8
https://portal.minimal-gaia-x.eu/asset/did:op:ae5D24b8ED7932F2146C0F4baf1D7Be8fF4F13C4
https://portal.minimal-gaia-x.eu/asset/did:op:3072145E0fDb31d1Aee26d985CA8873c1eB68088
